### PR TITLE
[B5] complete the split for `hqwebapp`

### PR DESCRIPTION
**Pull request by biyeun**
## Technical Summary
Lots of improvements to `migrate_app_to_bootstrap5` have been made since the original split of `hqwebapp`. This PR introduces additional file splits necessary for `hqwebapp`, as well as some small cleanups.

It also fixes the mobile ux warning for bootstrap 5 pages:

bootstrap3 version:
Screen Shot 2024-05-01 at 10 42 00 PM
Screen Shot 2024-05-01 at 10 42 09 PM

bootstrap5 version:
Screen Shot 2024-05-01 at 10 42 58 PM
Screen Shot 2024-05-01 at 10 41 41 PM

### Safety story
Mostly renames of files. Only change to active pages is the mobile ux warning, which was tested extensively locally.

### Automated test coverage
diffs are covered by tests

### QA Plan
Not needed

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_modal.scss [hunk 1]
```diff
@@ -57,31 +57,6 @@
     @include absolute-center-modal(20px);
 }
 
-// Modal that has some kind of control (pagination, search box, etc.) in header or footer
-.modal-header.with-controls, .modal-footer.with-controls {
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_modal.scss:77]: These are used in [data_corrections_modal.html](https://github.com/dimagi/commcare-hq/blob/d1d9a20e6cb4a808a494b8e24478cd342dda5f99/corehq/apps/reports/templates/reports/partials/data_corrections_modal.html) and [case_property_modal.html](https://github.com/dimagi/commcare-hq/blob/d1d9a20e6cb4a808a494b8e24478cd342dda5f99/corehq/ex-submodules/casexml/apps/case/templates/case/partials/case_property_modal.html).

File: corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/bootstrap5/multifield.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% load crispy_forms_field %}
+
+{% if field.is_hidden %}
+    {{ field }}
+{% else %}
+    {% crispy_field field %}  {# todo B5: check crispy #}
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/bootstrap5/multifield.html]: I don't think this template needs to be split.

Comment (orangejenny) [reply corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/bootstrap5/multifield.html]: Nevermind, this got reverted. It'd be helpful to run a rebase before creating the PR.

**Comment by orangejenny** [comment]
Thank you for the crispy split. In the settings migration, I've been running into form errors being displayed unstyled, and this gives me direction on where to fix those.

File: corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_modal.scss (hunk 1 above)
Comment (biyeun) [reply corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_modal.scss:77]: ah i was grepping for the styles WITH the `.`, going to revert

Comment (biyeun) [reply corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_modal.scss:77]: reverted

File: corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap5/create_item_form.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+{% load crispy_forms_tags %}
+{% crispy form %}  {# todo B5: check crispy #}
```
[the remark this answers is not in the document]
Comment (biyeun) [reply corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap5/create_item_form.html]: reverted. missed that one, thanks

**Comment by biyeun** [comment]
@orangejenny issues addressed ^_^

**Review by orangejenny — APPROVED** [review]

**Review by orangejenny — APPROVED** [review]

**Comment by biyeun** [comment]
@orangejenny figured it out...`mocha/base.html` didn't know what version of bootstrap to use

**Review by orangejenny — APPROVED** [review]

**Review by orangejenny — APPROVED** [review]

**Pull request merged**